runtime-modules/content/src/lib.rs

```diff
@@ -2540,8 +2540,8 @@ decl_event!(
         AuctionBidCanceled(MemberId, VideoId),
         AuctionCanceled(ContentActor, VideoId),
         EnglishAuctionCompleted(MemberId, VideoId),
-        BidMadeCompletingAuction(MemberId, VideoId, Option<MemberId>),
```

Comment:
I don't think this `Option` should be removed here?

Reply:
I was used an outdated version of `upstrea/rhodes`
Now it is fixed